test/libsolidity/solidityExecutionFramework.h

```diff
@@ -45,62 +45,22 @@ namespace solidity
 namespace test
 {
 
-std::string const c_genesisConfiguration = R"(
-{
-	"sealEngine": "NoProof",
-	"options": {
-		"authorities": [
-			"0xfa0c706a1410c8785baa7498325cf7461b325583",
-			"0x7766d151b2c63cb096f624daa091ccb27a2c693f"
-		]
-	},
-	"params": {
-		"accountStartNonce": "0x",
-		"maximumExtraDataSize": "0x1000000",
-		"blockReward": "0x",
-		"registrar": "",
-		"networkID" : "0x25",
-	"difficulty" : "0x13141"
-	},
-	"genesis": {
-		"author": "0x0000000000000000000000000000000000000000",
-		"timestamp": "0x00",
-		"parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-		"extraData": "0x",
-		"gasLimit": "0x1000000000000"
-	},
-	"accounts": {
-		"0000000000000000000000000000000000000001": { "wei": "1", "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } } },
-		"0000000000000000000000000000000000000002": { "wei": "1", "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
-		"0000000000000000000000000000000000000003": { "wei": "1", "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
-		"0000000000000000000000000000000000000004": { "wei": "1", "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } }
-	},
-	"network": {
-		"nodes": [
-				 ]
-	}
-}
-)";
-
 class ExecutionFramework
 {
 
 public:
 	ExecutionFramework():
-		m_state(0),
-		m_socket("/media/www/STUFF/Ethereum/testnet/ethnode1/datadir/geth.ipc")
+		m_state(0)
+		//m_socket("/home/wins/Ethereum/testnet/ethnode1/geth.ipc") //test
```

Comment:
We either have to start the node on our own and determine a random tmp path or the path needs to be passed in via an environment variable. Probably best to allow both.